tests/build/test_build_request.py

```diff
@@ -57,6 +58,21 @@ def test_build_request_isnt_auto_instantiated(self):
         flexmock(br).should_receive('template').and_return(build_json)
         assert br.is_auto_instantiated() is False
 
+    def test_set_label(self):
+        build_json = copy.deepcopy(TEST_BUILD_JSON)
+        br = BuildRequest('something')
+        flexmock(br).should_receive('template').and_return(build_json)
+        assert br.template['metadata'].get('labels') is None
+
+        br.set_label('label-1', 'value-1')
+        br.set_label('label-2', 'value-2')
+        br.set_label('label-3', 'value-3')
+        assert br.template['metadata']['labels'] == {
+                'label-1': 'value-1',
```

Comment:
'pep8' doesn't like this indent:
`E126 continuation line over-indented for hanging indent`